eng/pipelines/templates/Build_and_UnitTest.yml

```diff
@@ -47,7 +47,7 @@ steps:
 - task: MicroBuildSwixPlugin@1
   displayName: "Install Swix Plugin"
 
-- task: ms-vseng.MicroBuildTasks.965C8DC6-1483-45C9-B384-5AC75DA1F1A4.MicroBuildOptProfPlugin@6
+- task: MicroBuildOptProfPlugin@6
```

Comment:
Does this `@6` have any impact on build? Looks like some numbering. In general can we give any name appropriate for the context or it's already decided somewhere else?

Reply:
https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=schema%2Cparameter-schema#task

> `- task: string  # reference to a task and version, e.g. "VSBuild@1"`